cypress/integration/admin_portal/pages/enterprise_code_management.js

```diff
@@ -28,6 +28,62 @@ class CodeManagementPage {
     cy.get('.request-codes-btn.btn.btn-primary').click()
   }
 
+  getCoupon() {
+    return cy.get('.coupon')
+  }
+
+  getCouponDetailsRow() {
+    return cy.get('.expanded .metadata.row')
+  }
+
+  getCouponDetailsRowText() {
+      return cy.get('small.text-light')
+  }
+
+  getFilterOptions(){
```

Comment:
we have placed white spaces in above functions before curly brackets, we should follow the same pattern throughout